stan/math/fwd/fun/mdivide_left_ldlt.hpp

```diff
@@ -13,27 +13,27 @@ namespace math {
 /**
  * Returns the solution of the system Ax=b given an LDLT_factor of A
  *
- * @tparam R1 number of rows in the LDLT_factor, can be Eigen::Dynamic
- * @tparam C1 number of columns in the LDLT_factor, can be Eigen::Dynamic
+ * @tparam T type of matrix for the LDLT_factor
  * @tparam EigMat type of the right-hand side matrix or vector
- *
  * @param A LDLT_factor
  * @param b right-hand side matrix or vector
  * @return x = b A^-1, solution of the linear system.
  * @throws std::domain_error if rows of b don't match the size of A.
  */
-template <int R1, int C1, typename EigMat,
+template <typename T, typename EigMat,
+          require_eigen_vt<std::is_arithmetic, T>* = nullptr,
```

Comment:
I think technically it needs to be like this if the `LDLT_factor` got constructed with something that has a plain type matrix but isn't actually a matrix itself. But since I push everything through `make_ldlt_factor` now I already do the conversion to plain type.

Anyway I'm pretty sure if I changed the signature to `LDLT_factor<Eigen::MatrixXd>` it would work.